### PR TITLE
fix: add documentation of UserInfo policy

### PR DESCRIPTION
**Pull request by ytvnr**
---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `1.5.2-add-documentation-SNAPSHOT`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/policy/gravitee-policy-openid-connect-userinfo/1.5.2-add-documentation-SNAPSHOT/gravitee-policy-openid-connect-userinfo-1.5.2-add-documentation-SNAPSHOT.zip)

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -20,3 +20,17 @@ endif::[]
 |===
 
 == Description
+
+You can use the `policy-openid-userinfo` to get the OpenId Connect user info from an OAuth2 resource through its UserInfo endpoint.
+
+NOTE: If no Oauth2 resource is defined for the policy, or it is not well configured, the request will fail with a 401 status. More information will be available in `WWW_Authenticate` header
```
Comment (mouligno) [inline README.adoc:26]:
```suggestion
NOTE: If no Oauth2 resource is defined for the policy, or misconfigured, the request will fail with a 401 status. More information will be available in `WWW_Authenticate` header
```

**Review by mouligno — CHANGES_REQUESTED** [review]

**Comment by mouligno** [comment]
@ivan-gravitee any thoughts on this documentation ?

File: README.adoc (hunk 1 above)
Comment (ivan-gravitee) [inline README.adoc:26]:
```suggestion
NOTE: The request will fail with a 401 status If the policy's Oauth2 resource is misconfigured or not defined at all. To troubleshoot this, check the `WWW_Authenticate` header for more information.
```

Comment (ivan-gravitee) [reply README.adoc:26]: please see the other variant below

File: README.adoc [hunk 2]
```diff
@@ -20,3 +20,17 @@ endif::[]
...
+
+== Configuration
+
+You can configure the policy with the following options:
```
Comment (ivan-gravitee) [inline README.adoc:30]:
```suggestion
Use the following options to configure the policy:
```

File: README.adoc [hunk 3]
```diff
@@ -20,3 +20,17 @@ endif::[]
 |===
 
 == Description
+
+You can use the `policy-openid-userinfo` to get the OpenId Connect user info from an OAuth2 resource through its UserInfo endpoint.
```
Comment (ivan-gravitee) [inline README.adoc:24]:
```suggestion
Use the `policy-openid-userinfo` to get the OpenId Connect user info from an OAuth2 resource through its `UserInfo` endpoint.
```

File: README.adoc [hunk 4]
```diff
@@ -20,3 +20,17 @@ endif::[]
...
+
+== Configuration
+
+You can configure the policy with the following options:
+
+|===
+|Property |Required |Description |Type |Default
+
+|oauthResource|X|The OAuth2 resource used to get UserInfo|string|
+|extractPayload||When `true`, the payload of the response from UserInfo endpoint is set in the gateway `openid.userinfo.payload` attribute|boolean|
```
Comment (ivan-gravitee) [inline README.adoc:36]:
```suggestion
|extractPayload||When set to `true`, the payload of the response from the UserInfo endpoint is set in the `openid.userinfo.payload` gateway attribute|boolean|
```

**Review by ivan-gravitee — CHANGES_REQUESTED** [review]
Looks good - added a few small suggestions.

**Review by ivan-gravitee — APPROVED** [review]
LGTM

**Pull request merged**